packages/checkout/sdk/src/smartCheckout/sell/sell.ts

```diff
@@ -130,18 +131,19 @@ export const sell = async (
     orderbook = instance.createOrderbookInstance(config);
     const { seaportContractAddress } = orderbook.config();
     spenderAddress = seaportContractAddress;
-    const sellItem = sellToken.type === ItemType.ERC721
+
+    const sellItem = 'type' in sellToken && sellToken.type === ItemType.ERC1155
```

Comment:
Not a blocker.
This can benefit from a coercion op `sellToken?.type === ItemType.ERC1155`

Reply:
From what I can tell, the type guard is required because the deprecated ERC721 item type (which this PR restores support for) does not include a type property, so to explicit type guard is required, and we know the ERC1155 variant is guaranteed to have the property
<img width="1119" alt="Screenshot 2024-05-20 at 10 25 32 am" src="https://github.com/immutable/ts-immutable-sdk/assets/11344927/6aefb065-59e9-4d82-97b2-63b577872154">

Reply:
Makes sense thanks, Missed the deprecated type line